tap_recharge/client.py

```diff
@@ -184,11 +184,10 @@ def request(self, method, path=None, url=None, **kwargs): # pylint: disable=too-
             kwargs['headers'] = {}
         kwargs['headers']['X-Recharge-Access-Token'] = self.__access_token
         kwargs['headers']['Accept'] = 'application/json'
-        # If we did not specify any API Version during API Call, the Recharge will use the default API Version of our store
-        # the 'collections' was added as part of API Version: '2021-11', for older API Version,
-        # we will get empty records so adding 'X-Recharge-Version' for 'collections' API call
-        if path == 'collections':
-            kwargs['headers']['X-Recharge-Version'] = '2021-11'
+        kwargs['headers']['X-Recharge-Version'] = '2021-11'
+        # Products and Shop are not supported in new version: 2021-11
+        if path in ['products', 'shop']:
```

Comment:
We can remove the `shop` from here as it is replaced with a `store` in another PR.

Reply:
When we will remove **shop** from here, the Tap will use 2021-11 for getting shop records. We will get 404 Not Found error:
> {
>     "error": "Not Found"
> }